### PR TITLE
Make markdown files in .github folder

### PR DESCRIPTION
**Pull request by sarahdactyl71**
### What does this do and why?
https://urbanairship.atlassian.net/browse/LIBS-116

Updates the repository to have built in markdown files for things like pull requests, and contributing to the repo. 

### Additional notes for reviewers
Please check that the information is accurate e.g. it says "Airship" vs "Urban Airship". Also please make note of any spelling or grammar mistakes. 

### Testing
Not a very easy way to test unless you have a markdown viewer. 

* I've tested in virtual environments for python versions:

- [ ] 2.7
- [ ] 3.5
- [ ] 3.6
- [ ] 3.7

### CODE REVIEW

**Review by devinsmythe — APPROVED** [review]

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+2.
+3.
+4.
+
+## Detailed Description
+<!--- Provide a detailed description. If possible full logs showing the behavior.-->
+
+## Possible Fix
+<!--- Not required, but suggest your possible solution if possible. -->
+<!--  If your fix is known to work, we accept pull requests with a completed contribution agreement. -->%
```
Comment (krubasaur) [inline .github/ISSUE_TEMPLATE.md]: >`<!--  If your fix is known to work, we accept pull requests with a completed contribution agreement. -->%`

Not sure if this was a necessary/intentional `%` in line or a typo

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+- [ ] 2.7
+- [ ] 3.5
+- [ ] 3.6
+- [ ] 3.7
+
+### Urban Airship Contribution Agreement
+[Link here](https://docs.google.com/forms/d/e/1FAIpQLScErfiz-fXSPpVZ9r8Di2Tr2xDFxt5MgzUel0__9vqUgvko7Q/viewform)
+
+- [ ] I've filled out and signed UA's contribution agreement form
```
Comment (krubasaur) [inline .github/PULL_REQUEST_TEMPLATE.md:21]: Update to "Airship's contribution agreement form" instead of "UA's"

File: .github/SUPPORT.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# Support Requests
+
+All requests for support including implementation support and feature requests are made to the Urban Airship support team.
```
Comment (krubasaur) [inline .github/SUPPORT.md]: >All requests for support including implementation support and feature requests are made to the Urban Airship support team.

update adding comments?

>All requests for support, including implementation support and feature requests, are made to the Urban Airship support team.

Comment (krubasaur) [reply .github/SUPPORT.md]: Oh and it references Urban Airship support team. Change this to Airshp :)

**Review by krubasaur — CHANGES_REQUESTED** [review]
I left a few comments about UA references and some grammar suggestions.

File: .github/SUPPORT.md (hunk 3 above)
Comment (sarahdactyl71) [reply .github/SUPPORT.md]: @krubasaur What did you mean exactly about adding comments?

Comment (krubasaur) [reply .github/SUPPORT.md]: Commas* 🤦🏾‍♀️sorry

**Review by krubasaur — APPROVED** [review]
:octocat: 🎉

**Pull request merged**